ui/pages/configuration/filters.js

```diff
@@ -0,0 +1,68 @@
+import { NoSsr, Paper, withStyles } from "@material-ui/core";
+import MesheryFilters from "../../components/Filters";
+import { updatepagepath } from "../../lib/store";
+import { connect } from "react-redux";
+import { bindActionCreators } from 'redux'
+import Head from 'next/head';
+import { getPath } from "../../lib/path";
+import { resolveHref } from "next/dist/next-server/lib/router/router";
```

Comment:
You can comment this line i.e Line 8 as it is unused across the file. This will clear the UI Integration test